content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -34,7 +34,7 @@ Network policies are implemented by the [network plugin](/docs/concepts/extend-k
 
 ## The Two Sorts of Pod Isolation
 
-There are two sorts of isolation for a pod: isolation for egress, and isolation for ingress.  They concern what connections may be established.  "Isolation" here is not absolute, rather it means "some restrictions apply".  The alternative, "non-isolated for $direction", means that no restrictions apply in the stated direction.  The two sorts of isolation (or not) are declared independently, and are both relevant for a connection from one pod to another.
+There are two sorts of isolation for a pod: isolation for egress, and isolation for ingress.  They concern what connections may be established.  "Isolation" here is not absolute, rather it means "some restrictions apply".  The alternative, "non-isolated for direction", means that no restrictions apply in the stated direction.  The two sorts of isolation (or not) are declared independently, and are both relevant for a connection from one pod to another.
```

Comment:
The `$` is there to imply that direction is a variable / a placeholder. If we remove the `$` (which is reasonable), we need to find a different way to make it clear that “direction” is a placeholder. For example: write “direction” in italics.

PS. It might be useful to leave in a note to localizers about intent here, as not all writing systems support italic-style emphasis.

Reply:
I have read it a couple of times, and have following suggestions to better demonstrate it is a placeholder:
Instead of "non-isolated for $direction", you say "non-isolated for ingress/egress".
Or simply dropping the `$`. I did not find an example of _italic_ in the page. so it might to match the style of document.

Reply:
given that the sentence already clarifies "in the stated direction" immediately afterward, I think you could just remove "`for $direction`" entirely

```suggestion
There are two sorts of isolation for a pod: isolation for egress, and isolation for ingress.  They concern what connections may be established.  "Isolation" here is not absolute, rather it means "some restrictions apply".  The alternative, "non-isolated", means that no restrictions apply in the stated direction.  The two sorts of isolation (or not) are declared independently, and are both relevant for a connection from one pod to another.
```